dm_control/locomotion/walkers/assets/dog_v2/build_dog.py

```diff
@@ -33,20 +37,34 @@
 from dm_control.utils import io as resources
 
 flags.DEFINE_boolean('make_skin', True, 'Whether to make a new dog_skin.skn')
+flags.DEFINE_boolean('lengthrange_from_joints', False,
+                     'Whether to compute the length ranges of spatial tendons from joints')
+flags.DEFINE_boolean('use_muscles', False,
+                     'Whether to use muscles or torque actuators')
+flags.DEFINE_boolean('add_markers', False,
+                     'Whether to add markers; used for motion capture tracking')
+flags.DEFINE_integer(
+  'muscle_strength_scale', -1,
+  'muscle force multiplier, if negative we let MuJoCo computer the strength')
+flags.DEFINE_string(
+  'muscle_dynamics', 'Millard',
+  'muscle dynamics one of [Millard, Sigmoid, General]')
+flags.DEFINE_boolean(
+  'make_muscles_skin', False, 'Whether to make new skins for muscles')
 flags.DEFINE_float(
-    'lumbar_dofs_per_vertebra',
-    1.5,
-    'Number of degrees of freedom per vertebra in lumbar spine.',
+  'lumbar_dofs_per_vertebra',
```

Comment:
Please revert these indentation changes since they are not necessary for your PR and are at odds with Google's formatting style.